schema/README.md

```diff
@@ -24,8 +24,8 @@ To build it:
 
 ```bash
 export GOPATH=`mktemp -d`
```

Comment:
Maybe we should add it here instead?

```suggestion
export GOPATH=`mktemp -d` GO111MODULE=off
```

Reply:
@tianon I have updated as your suggestion.

BTW Makefile has the same issue but GitHub workflow calls `go mod init` before calling `make validate`.